appcheck/app/src/main/java/com/google/firebase/example/appcheck/kotlin/ApiWithAppCheckExample.kt

```diff
@@ -20,14 +23,19 @@ class ApiWithAppCheckExample {
         .build()
         .create(YourExampleBackendService::class.java)
 
-    fun callApiExample() {
-        FirebaseAppCheck.getInstance()
-            .getAppCheckToken(false)
-            .addOnSuccessListener { tokenResponse ->
-                val appCheckToken = tokenResponse.token
-                val apiCall = yourExampleBackendService.exampleData(appCheckToken)
-                // ...
-            }
+    suspend fun callApiExample() {
+        val tokenResponse = Firebase.appCheck.getToken(false).await()
```

Comment:
It seems like the current snippet is calling `getAppCheckToken(false)` and the one on this PR is calling `getToken(false)`. Is this change intended?